README.markdown

```diff
@@ -136,35 +114,51 @@ This will ensure the version `1.4.0` of the puppet-agent package is installed. F
 
 ##### `arch`
 
-The architecture version you wish to install. Defaults to `$::architecture`. This parameter is [ignored](#known-issues) in Windows Server 2003.
+The architecture version you wish to install. Defaults to `$::facts['architecture']`.
+``` puppet
+  arch => 'x86_64'
+```
 
 ##### `collection`
 
-The Puppet Collection to track. Defaults to `PC1`.
+The Puppet Collection to track, should be one of `puppet5` or `puppet6`. Puppet collections contain the latest agents included
+in the collection's series, so the latest 5 series in puppet5 (for example: 5.5.10) and the latest 6 series in puppet6 (for
```

Comment:
I think these specific examples may become confusing later, since they won't match the latest versions anymore -- maybe we could say, like `the latest 5.y.z` or something like that? Don't know if it actually matters that much though